lib/div-alg.gi

```diff
@@ -1631,6 +1631,10 @@ if p < q then
   q:=a;
 fi;
 
+if q<-1 or (q=-1 and not(p=-1 or IsPrimeInt(p))) or (q>-1 and not(IsPrimeInt(q))) or (q>-1 and IsPrimeInt(q) and not(IsPrimeInt(p))) then
+return "Invalid, both inputs must be -1 or positive prime integers.";
```

Comment:
But this shouldn't this be an error because the function is called with wrong arguments, instead of returning a string? 

Also, we do not put dot at the end of a returned string or an error message, since they may be part of a larger text.

Finally, you can also test error messages, but note that the test file contains only the initial part of the backtrace -see examples at https://github.com/gap-system/gap/blob/master/tst/testinstall/boolean.tst

Reply:
The issue is that it makes perfect sense mathematically to ask for the local indices of a symbol algebra like (-2,6/Q), but the LocalIndicesOfRationalQuaternionAlgebra function is designed for that.  This function was accepting invalid input and returning incorrect lists of local indices, so that is what I want to avoid.  I just felt it wouldn't be necessary to declare an error and cause a break loop in this case, just a warning that explains why the function isn't giving an answer would accomplish what is needed.

Reply:
Returning a string is still quite contrary to the style used in GAP. What it it will return `fail` and a warning will be displayed using `Info` mechanism?